mysite/account/views.py

```diff
@@ -413,7 +413,7 @@ def invite_someone_do(request):
 # modified trivially in the POST handler.
 
 
-@django_authopenid.views.not_authenticated
+@authenticated
```

Comment:
@paulproteus should this be `@django_authopenid.views.authenticated` or is `@authenticated` sufficient